.github/merge_rules.yaml

```diff
@@ -85,7 +85,7 @@
   - EasyCLA
   - Lint
   - pull / linux-bionic-py3_8-clang8-xla / build
-  - pull / linux-bionic-py3_8-clang8-xla / test (xla, 1, 1, linux.2xlarge)
+  - pull / linux-bionic-py3_8-clang8-xla / test (xla, 1, 1, linux.4xlarge)
```

Comment:
Or we could try to figure out how to make `pull / linux-bionic-py3_8-clang8-xla / test (xla)` works.  That's the convention I use to disable job, calculate reliability and TTS metrics, filter jobs, etc..